.s2i/httpd-cfg/01-commercial.conf

```
@@ -696,6 +696,10 @@ AddType text/vtt                            vtt
     RewriteRule ^dedicated/osd_private_connections/?(.*)$ /dedicated/osd_cluster_admin/osd_private_connections/$1 [NE,R=301]
 
     RewriteRule ^dedicated/nodes/rosa-managing-worker-nodes.html /dedicated/osd_cluster_admin/osd_nodes/osd-managing-worker-nodes.html [NE,R=301]
+    
+    # Redirect to get Custom Metrics Autoscaler into OSD
+    RewriteRule ^dedicated/nodes/cma/?(.*)$ /dedicated/nodes/cma/$1 [NE,R=301]
```

```suggestion
    RewriteRule ^dedicated/nodes/cma/?(.*)$ - [L,R=302]
```
This rule should allow everything under the `cma` directory to be excluded from the catch-all redirect on the next line.